Mathlib/Analysis/InnerProductSpace/PiL2.lean

```diff
@@ -151,19 +151,17 @@ theorem EuclideanSpace.sphere_zero_eq {n : Type*} [Fintype n] (r : ℝ) (hr : 0
   simp_rw [mem_setOf, mem_sphere_zero_iff_norm, norm_eq, norm_eq_abs, sq_abs,
     Real.sqrt_eq_iff_sq_eq this hr, eq_comm]
 
-variable [Fintype ι]
-
 section
 
--- Porting note: no longer supported
--- attribute [local reducible] PiLp
-
-instance EuclideanSpace.instFiniteDimensional : FiniteDimensional 𝕜 (EuclideanSpace 𝕜 ι) := by
-  infer_instance
+instance EuclideanSpace.instFiniteDimensional [Finite ι] :
+    FiniteDimensional 𝕜 (EuclideanSpace 𝕜 ι) :=
+  inferInstance
 #align euclidean_space.finite_dimensional EuclideanSpace.instFiniteDimensional
 
-instance EuclideanSpace.instInnerProductSpace : InnerProductSpace 𝕜 (EuclideanSpace 𝕜 ι) := by
-  infer_instance
+variable [Fintype ι]
+
+instance EuclideanSpace.instInnerProductSpace : InnerProductSpace 𝕜 (EuclideanSpace 𝕜 ι) :=
+  inferInstance
 #align euclidean_space.inner_product_space EuclideanSpace.instInnerProductSpace
```

Comment:
I'm not sure I get the point of these instances if they can already be derived automatically. Why not just remove them?